### ISSUE TITLE
Login form layout

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#1069]
The remember me checkbox comes after the submit button, which is problematic for screen reader users (i.e. they might never realize it’s there).

During the a11y review meeting, we discussed some options:

- Update the layout so that the checkbox is before/above the submit button
- Update the dom order so that the checkbox is before the submit button, but use css to keep the same layout as we have now.

The first option seemed to be the preferred approach, but we want to run this by design first. If design feels that the change in layout is not preferred, then we can proceed with the second option.

**Comment by mceledonia** [comment]
@jknilans and I were just discussing this. The login form is also missing a "Forgot Password?" text button. We could move the "Remember me" checkbox above the submit button, and right-align the "Forgot password?" button text on the same line.

**Comment by jknilans** [comment]
@rachael-phillips I've been working with @mceledonia on an update to the login page. We have a few changes we'd like to propose that would make the current PF4 version more consistent with the PF3 version.

**Screenshot of changes made in the browser:**
<img width="1205" alt="screen shot 2019-01-07 at 10 44 11 am" src="https://user-images.githubusercontent.com/25486044/50777323-403b3980-1269-11e9-95ab-d35cab624a3c.png">

**Video comparing current and proposed PF4 Login pages:**
![login_mods](https://user-images.githubusercontent.com/25486044/50777911-c6a44b00-126a-11e9-8fcf-faa91e3bc408.gif)

Updates include:

- moving language selector to right side
- move "need an account?" link to the bottom of the page (the main task at a login page is to log in, ot to create an account; this emphasizes the main task and de-emphasizes a secondary task of creating an account)
- added "Forgot password?" link to Password input field
- moved "keep me logged in" to above the submit button (order of operation is select box then click button, this change fits the mode of operation)
- full width login button corresponds visually with SSO/multi-factor where user may only view one input field at a time.
- removed company names from other ways to login

Thoughts @kybaker @mcarrano ?

**Comment by mcarrano** [comment]
@jknilans @mceledonia I think that the changes you propose here make sense from a usability perspective. My only question about the design is it looks like we've added some sort of footer at the bottom where the "Need an account? Sign up" sits. Is that a new element?  If so, I can see potential implementation concerns with that and would suggest running by a CSS developer.

Before finalizing this, I would also ask you to consider how server side validation errors and other notifications are reported as there were open issues around those. We may as well address everything at once.

**Comment by amysueg** [comment]
@junezhang for awareness

**Comment by matthewcarleton** [comment]
Hey @mceledonia @jknilans I'm looking at the sketch file and seeing some differences between here and there. Can you confirm which version is finalized?

**Comment by mceledonia** [comment]
Hey @matthewcarleton thanks for the heads up, the sketch file versions are the latest. I will upload some screenshots here.
